root/components/PartialDateInput.js

```diff
@@ -159,7 +156,6 @@ const PartialDateInput = ({
         size={4}
         type="text"
         {...yearProps}
-        {...inputProps}
```

Comment:
I haven't found anywhere that we use this.  But if we do, I think it's better we define the specific props we can pass through.

Reply:
I tried, but Flow also doesn't like this. :(  I'll keep it as-is for now...

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ root/components/FormRowPartialDate.js:63:6

Cannot create PartialDateInput element because: [incompatible-type]
 • Either undefined [1] is incompatible with function type [2] in property dispatch.
 • Or function type [1] is incompatible with undefined [3] in property dispatch.

     root/components/FormRowPartialDate.js
 [1] 51│   dispatch,
       :
     60│       label={label}
     61│       required={required}
     62│     />
     63│     <PartialDateInput
     64│       disabled={disabled}
     65│       dispatch={dispatch}
     66│       field={field}

     root/components/PartialDateInput.js
 [2] 33│       +dispatch: (ActionT) => void,
       :
 [3] 38│       +dispatch?: void,



Found 1 error
```